interface/patient_file/summary/immunizations.php

```diff
@@ -343,497 +346,474 @@ function saveImmunizationObservationResults($id, $immunizationdata)
-            <?php echo generate_select_list('immuniz_completion_status', 'Immunization_Completion_Status', $immuniz_completion_status, 'Select Completion Status', ' ');?>          </td>
-        </tr>
-        <tr>
-          <td align="right" class='text'>
-                <?php echo xlt('Substance Refusal Reason'); ?>
-          </td>
-          <td>
-            <?php echo generate_select_list('immunization_refusal_reason', 'immunization_refusal_reason', $immuniz_refusal_reason, 'Select Refusal Reason', ' ');?>
-          </td>
-        </tr>
-        <tr>
-          <td align="right" class='text'>
-                <?php echo xlt('Immunization Ordering Provider'); ?>
-          </td>
-          <td>
-            <select name="ordered_by_id" id='ordered_by_id'>
-            <option value=""></option>
+<body>
+    <div class="container mt-3">
+        <div class="row">
+            <div class="col-12">
+                <h2><?php echo xlt('Immunizations'); ?></h2>
+            </div>
+            <div class="col-12">
                 <?php
-                $sql = "select id, CONCAT(IFNULL(lname,''), ' ,',IFNULL(fname,'')) as full_name " .
-                       "from users where username != '' and password != 'NoLogin' " .
-                       "order by full_name";
-
-                $result = sqlStatement($sql);
-                while ($row = sqlFetchArray($result)) {
-                    echo '<OPTION VALUE=' . attr($row['id']);
-                    echo (isset($ordered_by_id) && $ordered_by_id != "" ? $ordered_by_id : $_SESSION['authUserID']) == $row['id'] ? ' selected>' : '>';
-                    echo text($row['full_name']) . '</OPTION>';
-                }
+                    $list_id = ""; // to indicate nav item is active, count and give correct id
+                    $menuPatient = new PatientMenuRole();
+                    $menuPatient->displayHorizNavBarMenu();
                 ?>
-            </select>
-          </td>
-        </tr>
-    <?php
-    if ($entered_by) {
-        ?>
-    <tr>
-        <td align="right" class='text'>
-            <?php echo xlt('Entered By'); ?>
-         </td>
-        <td>
-            <?php echo text($entered_by); ?>
-        </td>
-    </tr>
-        <?php
-    }
+            </div>
+            <div class="col-12">
+                <form class="jumbotron p-4" action="immunizations.php" name="add_immunization" id="add_immunization">
+                    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+
+                    <input type="hidden" name="mode" id="mode" value="add" />
+                    <input type="hidden" name="id" id="id" value="<?php echo attr($id); ?>" />
+                    <input type="hidden" name="pid" id="pid" value="<?php echo attr($pid); ?>" />
+                    <input type="hidden" name="uuid" id="uuid" value="<?php echo attr($uuid); ?>" />
 
-    if ($GLOBALS['observation_results_immunization']) {
```

Comment:
this if observation_results_immunization conditional needs to stay around the `Click here to see observation results` line

Reply:
hi @stu01509 , otherwise code looks good. just let me know when above is addressed and then I'll do some quick testing and then bring it in.

Reply:
Thanks for your mentions, update the commit :)